File: Recursion/sum_of_square.c
#include <stdio.h>

int sumOfSquares(int n) {
    if (n == 1)
        return 1;
    else
        return (n * n) + sumOfSquares(n - 1);
}

int main() {
    int N;
    printf("Enter the value of N: ");
    scanf("%d", &N);
    int result = sumOfSquares(N);
    printf("Sum of squares: %d\n", result);
    return 0;
}

File: Recursion/sum_of_square.py
def sum_of_squares(n):
    if n == 1:
        return 1
    else:
        return n**2 + sum_of_squares(n - 1)

N = int(input("Enter the value of N: "))
result = sum_of_squares(N)
print("Sum of squares:", result)
